src/main/java/picard/util/IntervalListTools.java

```diff
@@ -253,38 +253,68 @@
             IntervalList act(final List<IntervalList> list, final List<IntervalList> __) {
                 return IntervalList.concatenate(list);
             }
+
+            @Override
+            public boolean takesSecondInput() {
+                return false;
```

Comment:
If this is a boolean value that all enum instances must provide it would be simpler to model it as a boolean field in the enum, instead of a method.

Reply:
good point!